### PR TITLE
Various edits for clarity and grammar.

### PR DESCRIPTION
**Pull request by grahamgower**

### CODE REVIEW

File: popSimManu.tex [hunk 1]
```diff
@@ -399,36 +401,36 @@ \subsection*{Use case: comparing methods of demographic inference}
...
 \end{figure}
 
-\paragraph*{Multi Population demographic models}
+\paragraph*{Multi-population demographic models}
 As \texttt{stdpopsim} implements multi-population demographic models, we also
 explored parameter estimation of population divergence parameters. In particular
 we performed simulations under multi-population models for humans and \emph{Drosophila}
 and inferred simulated parameters using \dadi, \fastsimcoal, and \smcpp.
-For simplicity, we conducted inference in \dadi and \fastsimcoal using an IM model
+For simplicity, we conducted inference in \dadi and \fastsimcoal by fitting an island migration (IM) model
```
Comment (jeromekelleher) [inline popSimManu.tex]: IM stands for isolation with migration model I think. Perhaps we should cite a Jody Hey paper for reference.

**Comment by jeromekelleher** [comment]
LGTM.

**Comment by grahamgower** [comment]
Ok, I fixed IM and rebased.

**Comment by jeromekelleher** [comment]
Thanks -- LGTM.  I'll let @andrewkern merge.

Would you mind doing another quick rebase please @grahamgower, just in case there's any conflicts with #68?

**Comment by grahamgower** [comment]
No problem. I rebased again, no new conflicts.

File: popSimManu.tex (hunk 1 above)
Comment (andrewkern) [reply popSimManu.tex]: i can add appropriate citations once this is merged.

**Pull request merged**